File: AcademicWebApp/src/main/java/com/example/AcademicWebApp/Models/UserData.java
package com.example.AcademicWebApp.Models;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Data
@Entity(name = "user_data")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UserData {


    @Id
    @Column(name = "username")
    private String username;
    @Column(name = "name")
    private String name;
    @Column(name = "surname")
    private String surname;
    @Column(name = "email")
    private String email;
    @Column(name = "phone_number")
    private String phone_number;
    @Column(name = "home_address")
    private String home_address;
    @Column(name = "cnp")
    private String cnp;




}

File: DataGenerator/domain/userData.py

class UserData:

    def __init__(self, username, name, surname, email, phone, address, cnp):
        self.__username = username
        self.__name = name
        self.__surname = surname
        self.__email = email
        self.__phone = phone
        self.__address = address
        self.__cnp = cnp

    @property
    def username(self):
        return self.__username

    @property
    def name(self):
        return self.__name

    @property
    def surname(self):
        return self.__surname

    @property
    def email(self):
        return self.__email

    @property
    def phone(self):
        return self.__phone

    @property
    def address(self):
        return self.__address

    @property
    def cnp(self):
        return self.__cnp

    def __str__(self):
        return "'" + self.__username + "', '" + self.__name + "', '" + self.__surname + "', '" + self.__email + \
               "', '" + self.__phone + "', '" + self.__address + "', '" + self.__cnp + "'"
